README.md

```diff
@@ -49,7 +49,9 @@ Installation with CocoaPods
 |Name|Value|
 |:---|:----|
 |`FB_REFERENCE_IMAGE_DIR`|`$(SOURCE_ROOT)/$(PROJECT_NAME)Tests/ReferenceImages`|
+|`IMAGE_DIFF_DIR`|`$(SOURCE_ROOT)/$(PROJECT_NAME)Tests/FailureDiffs`|
 
+Define the `IMAGE_DIFF_DIR` to the directory where you want to store diffs of failed snapshots.
```

Comment:
Maybe move this few lines above and mark as point 3. Additional info about optional nature of this macro (and maybe default value) would be helpful.